app/views/contacts/_form.html.slim

```diff
@@ -0,0 +1,15 @@
+= form_for @contact do |f|
+
+  .form-group
```

Comment:
I'm not sure again if we need this. Normally the title pushes stuff down if it's for gaps. I guess I'm just asking to try without.

Reply:
Nice, thanks looks fine without so I'll take it out.